gems/datagrid/CVE-2019-14281.yml

```diff
@@ -0,0 +1,14 @@
+---
+gem: datagrid
+url: https://nvd.nist.gov/vuln/detail/CVE-2019-14281
+date: 2019-07-31
+title: Critical severity vulnerability that affects datagrid
+description: The datagrid gem 1.0.6 for Ruby, as distributed on RubyGems.org, included
+  a code-execution backdoor inserted by a third party.
+cvss_v3: 9.8
+patched_versions:
```

Comment:
Any reason why to keep the `patched_versions` part here? Maybe you mentioned it and I missed it somewhere... Versus just putting it all in `unaffected_versions`

Reply:
Adjusted for consistency.